static/app/utils/discover/discoverQuery.tsx

```diff
@@ -1,16 +1,19 @@
-import * as React from 'react';
+import {ReactNode, ReactText} from 'react';
```

Comment:
You don't even need to import this and can just use the global `React.*` namespace.

Any reason you prefer to import these?

Reply:
this is a nit btw. Need an eslint rule for this lol

Reply:
global imports 👁️ 👁️

Reply:
I don't really understand how it works lol, something to do with typescript global namespaces. There's some others that are in global namespaces like this

Reply:
It's because of the jsx transform update I'm pretty sure, still, I'm not a massive fan of global imports. I'm down to change it though.

Reply:
I figure when we _do_ need to import all the `React.*` types, it should be a pretty easy codemod

Reply:
Yeah, I think we should prob have robots enforce this w/ autofix if possible

Reply:
Follow up: https://github.com/getsentry/sentry/pull/33267